app/services/i18n/i18n.service.ts

```diff
@@ -0,0 +1,161 @@
+import electron from 'electron';
```

Comment:
Are you trying to introduce a new pattern here of naming services with `.service.ts`?  To me it seems a little redundant because it's already in `app/services` but if I'm missing something let me know.

Reply:
oh, I just have a homemade project with this style, so this is a mistake